packages/ra-ui-materialui/src/field/ReferenceArrayField.tsx

```diff
@@ -147,9 +149,9 @@ const useStyles = makeStyles(
     { name: 'RaReferenceArrayField' }
 );
 
-type ReferenceArrayFieldClassKey = 'progress';
+export type ReferenceArrayFieldClassKey = 'progress';
```

Comment:
I don't get this `progress` value for the key

Reply:
That's an old way to type the classesOverride for mui styling. Replaced with our own ClassesOverride